### ISSUE TITLE
Detect when numbers are used as keys in Mappings

### ISSUE THREAD

**Issue by hoegertn** [awslabs/cfn-python-lint#292]
*cfn-lint version: 0.5.1*

*Description of issue.*

Given the following template:

```
Mappings:
  AccountMap:
    # Dev
    123456789012:
      Some: Foo
    # Prod
    210987654321:
      Some: Bar
```

this should raise an error because the account ids have to be quoted.

**Comment by fatbasstard** [comment]
Quickly validated that... And you're completely right, good find! 👍 

```
Template validation error: Template format error: [/Mappings/AccountMap] map keys must be strings; received numeric [123456789012] instead ```

**Comment by hoegertn** [comment]
This issue can be closed, I think.

**Comment by cmmeyer** [comment]
👍Awesome! Thanks pointing it out.
